### ISSUE TITLE
something bad happen when I use autotuneValidation and pretrainedVectors simultaneously

### ISSUE THREAD

**Issue by BinNong** [facebookresearch/fastText#983]
raising Error：terminate without active exception

**Comment by Celebio** [comment]
Hi @BinNong ,
Thank you for reporting the issue.
Could you please provide the complete command line (changing the filenames if needed)?

Best regards,
Onur

**Comment by BinNong** [comment]
Thanks. I have fixed it. This exception is due to the dim of pre-trained vectors is not equal to the setting one
